helm/fiftyone-teams-app/README.md

```diff
@@ -779,7 +833,9 @@ A minimal example `values.yaml` may be found
 [ingress-default-ingress-class]: https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class
 [ingress-rules]: https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules
 [ingress]: https://kubernetes.io/docs/concepts/services-networking/ingress/
+[internal-auth-mode]: https://docs.voxel51.com/teams/pluggable_auth.html#internal-mode
 [labels-and-selectors]: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+[legacy-auth-mode]: topher/document-install-modal-override
```

Comment:
@findtopher - should the link point to https://docs.voxel51.com/teams/pluggable_auth.html#legacy-mode ?